target/linux/ramips/dts/mt7628an_tplink_re220-v2.dts

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7628an_tplink_re220.dtsi"
```

Comment:
If this really is a 100 % clone of the RE200 v2, and you didn't just copy everything without testing, then you might use the existing DTSI from the re200 here, as that's what the DTSI files are for (and drop your 1:1 DTSI copy).

Reply:
Yes, both devices share the same flash layout and GPIO pinout.